PKGBUILD

```diff
@@ -0,0 +1,24 @@
+pkgbase='python-ovos-config'
+pkgname=('python-ovos-config')
+_module='ovos-config'
+pkgver='0.0.7'
+pkgrel=1
+pkgdesc="OpenVoiceOS configuration tool"
+url="https://github.com/OpenVoiceOS/ovos-config"
+depends=('python')
```

Comment:
Package is missing the actual dependencies

Reply:
these would be dependencies already in AUR correct? so we need to start from ovos-utils and go from that one

but meanwhile if i understand right (just got started on docs) wouldn't line 23 `python setup.py install...` still install the packages? even if not best practice until we get other deps in the AUR

Reply:
There are build deps and runtime deps in arch pkgbuild, if a package requires build deps then those need to be available before setup.py itself runs

Reply:
these are pkgbuilds that I had generated for OVOS a while back, haven't been updated to the latest but should give an idea [ovos-core-pkgbuilds.tar.gz](https://github.com/OpenVoiceOS/ovos-config/files/11194718/ovos-core-pkgbuilds.tar.gz)